### PR TITLE
LRQA-71834 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-71834

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:relevant

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:relevant

File: modules/apps/account/account-test/src/testFunctional/macros/json/accountEntryUser/JSONAccountEntryUser.macro [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+			var creatorEmailAddress = "test@liferay.com";
+		}
+
+		if (!(isSet(creatorPassword))) {
+			var creatorPassword = "test";
+		}
+
+		if (!(isSet(accountEntryDescription))) {
+			var accountEntryDescription = "Account Entry Description";
+		}
```
Comment (patriciajeanperez) [inline modules/apps/account/account-test/src/testFunctional/macros/json/accountEntryUser/JSONAccountEntryUser.macro:16]: @melodywu probably not needed for all tests that use this macro to add a description to accounts

File: modules/apps/account/account-test/src/testFunctional/macros/json/accountEntry/JSONAccountEntryAPI.macro [hunk 2]
```diff
@@ -25,4 +25,44 @@ definition {
 		com.liferay.poshi.runner.util.JSONCurlUtil.post("${curl}");
 	}
 
+	@summary = "Helper function to get the 'accountEntryId' for the folder '${accountEntryName}'"
```
Comment (patriciajeanperez) [inline modules/apps/account/account-test/src/testFunctional/macros/json/accountEntry/JSONAccountEntryAPI.macro:28]: @melodywu make summary simpler

File: modules/apps/account/account-test/src/testFunctional/macros/json/accountEntryUser/JSONAccountEntryUser.macro (hunk 1 above)
Comment (melodywu) [reply modules/apps/account/account-test/src/testFunctional/macros/json/accountEntryUser/JSONAccountEntryUser.macro:16]: Done

File: modules/apps/account/account-test/src/testFunctional/macros/json/accountEntry/JSONAccountEntryAPI.macro (hunk 2 above)
Comment (melodywu) [reply modules/apps/account/account-test/src/testFunctional/macros/json/accountEntry/JSONAccountEntryAPI.macro:28]: Done

**Comment by patriciajeanperez** [comment]
Thanks @melodywu! You updated the test props but didn't run the tests though. I'll run to check

**Comment by patriciajeanperez** [comment]
I saw you ran them here? https://github.com/melodywu/liferay-portal/pull/27.
Please resend after fixing test and SF failures. You also want to make sure that you make a separate commit for temporarily updating the test properties and adding `property portal.acceptance = "melody";` to the tests. That way, they're easier to revert when you are ready to forward the test.

**Pull request closed without merging**